src/app/clusters/pump-configuration-and-control-server/pump-configuration-and-control-server.cpp

```diff
@@ -24,6 +24,7 @@
 //#include <app/CommandHandler.h>
 #include <app/ConcreteAttributePath.h>
 #include <app/ConcreteCommandPath.h>
+#include <app/InteractionModelEngine.h>
```

Comment:
Why is this being included?  I would not expect this to need the IM engine, and see nothing below that uses it at first glance.

Reply:
This is to get access to GetAttributeMetadata, is there another way to get access to that function?

Reply:
Ah, I see, OK.